app/config/msgraph.dev.configuration.yaml

```diff
@@ -0,0 +1,12 @@
+default:
+  tenantId: ${AUTH_MICROSOFT_TENANT_ID}
+  queryMode: advanced
+  user:
+    select: ['id', 'mail', 'displayName', 'description', 'userPrincipalName']
+    loadPhotos: false
+    filter: not(startsWith(userPrincipalName, 'a-')) and (endsWith(mail, 'defra.gov.uk') or endsWith(mail, 'defra.onmicrosoft.com'))
+  group:
+    filter: "displayName eq '${ADP_PORTAL_PLATFORM_ADMINS_GROUP}' or startsWith(displayName, '${ADP_PORTAL_USERS_GROUP_PREFIX}')"
+  schedule:
+    frequency: { hours: 0, minutes: 10 }
```

Comment:
If it's all users.. it can be 35 mins as we don't need dynamic refresh regularly

Reply:
Same with all of the AAD ones for pulling in 'all' users.